test/e2e/tests/risingwave/tests.sh

```diff
@@ -27,6 +27,11 @@ function test::risingwave::manifest_from() {
   envsubst "${@:2}" <"${manifest_file}"
 }
 
+function test::risingwave::enable_openkruise() {
+  logging::info "Enabling openkruise at Risingwave level"
+  k8s::kubectl patch risingwave -n "${E2E_NAMESPACE}"  "${E2E_RISINGWAVE_NAME}" --type merge -p '{"spec":{"enableOpenKruise":true}}'
```

Comment:
Try hiding the outputs in normal execution. 

```suggestion
  shell::run k8s::kubectl patch risingwave -n "${E2E_NAMESPACE}"  "${E2E_RISINGWAVE_NAME}" --type merge -p '{"spec":{"enableOpenKruise":true}}'
```